examples/smtp/envoy.yaml

```diff
@@ -0,0 +1,121 @@
+static_resources:
```

Comment:
@phlax what's our policy on contrib examples? Should they be in their own subdirectory so it's clear which ones are contrib and which aren't?

Reply:
sounds good, I didn't realize these were run programmatically by the ci so I didn't have any of that infrastructure